### ISSUE TITLE
get_camera_info(): res, cam_intrinsics, R

### ISSUE THREAD

**Issue by happysujin** [team-plask/wham-test#9]
res = emdb.camera.width, emdb.camera.height
cam_intrinsics = emdb.camera.intrinsics
R = (a function required?) emdb.camera.extrinsics (=labels['cam_poses'] maybe)
(from dataset_eval.py, EvalDataset: prepare_label:
R = self.labels['cam_poses'][index][:, :3, :3].clone()
cam_angvel = transforms.matrix_to_rotation_6d(R[:-1] @ R[1:].transpose(-1, -2))
cam_angvel = (cam_angvel - torch.tensor([[1, 0, 0, 0, 1, 0]]).to(cam_angvel)) * FPS
target['R'] = R)
